src/status_im/ui/screens/wallet/transactions/subs.cljs

```diff
@@ -33,11 +33,11 @@
 
 (defn format-unsigned-transaction [{:keys [id] :as transaction}]
   (assoc transaction
-    :type           :unsigned
-    :confirmations  0
-    :symbol         "ETH"
-    ;; TODO (andrey) revisit this, we shouldn't set not hash value to the hash field
-    :hash           id))
+         :type           :unsigned
+         :confirmations  0
+         :symbol         "ETH"
+         ;; TODO (andrey) revisit this, we shouldn't set not hash value to the hash field
+         :hash           id))
```

Comment:
I agree with these indentation changes, thanks.

However, it's actually not the first time I see that someone changed the proper indentation to the wrong one. Probably it happens because we have no common understanding of how indentation should be. So my suggestion is to start discussing it.

Reply:
This is actually a side-effect of saving the file with my current emacs config